static/css/navgar.css

```diff
@@ -0,0 +1,30 @@
+.navbar {
+  background: #ffffff;
+}
+
+.nav-link,
+.navbar-brand {
+  color: #fff;
+  cursor: pointer;
+}
+
+.nav-link {
+  margin-right: 1em !important;
+}
+
+.nav-link:hover {
+  color: #000;
+}
+
+.navbar-collapse {
+  justify-content: flex-end;
+  margin-right: -90mm;
```

Comment:
Just a suggestion, if you consider working with rem it will make your life much easier.
When using rem all the sizes are referenced from the root font size, thus help you achieve a more balanced design.
Furthermore, it makes your sizing more consistent - instead of using em, px, mm... you could just use rem.
You could learn more about it in this article - 
https://www.sitepoint.com/understanding-and-using-rem-units-in-css/ . 
Specifically read about how to make 1rem = 10px.

Reply:
thanks :)